### PR TITLE
Nerfs the meathook

### PR DESCRIPTION
**Pull request by Cyberboss**
Yes, I am the salty victim of hook + esword

Seriously though, it's really fun to get pulled from ESCAPE to MEDBAY and be insta dead

### CODE REVIEW

**Comment by Supermichael777** [comment]
might as well grab a fire extinguisher

**Comment by KorPhaeron** [comment]
Do a reasonable nerf instead of making it worse than a lasergun please

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -182,11 +182,11 @@
 	icon_state = "hook"
 	icon = 'icons/obj/lavaland/artefacts.dmi'
 	pass_flags = PASSTABLE
-	damage = 25
+	damage = 15
 	armour_penetration = 100
 	damage_type = BRUTE
 	hitsound = 'sound/effects/splat.ogg'
-	weaken = 3
+	weaken = 0
```
Comment (Iamgoofball) [inline code/modules/mining/lavaland/necropolis_chests.dm]: literally makes this worthless

nerf Phantom Lancer instead please

**Comment by Chnkr** [comment]
We need a "i ded" tag

**Comment by jaxy15** [comment]
Roadhog doesn't need a nerf

**Comment by ExcessiveUseOfCobblestone** [comment]
Make the projectile last within viewing range [7] if your concern is

>to get pulled from ESCAPE to MEDBAY and be insta dead

**Comment by KorPhaeron** [comment]
>being able to pull people a whole 7 tiles

boring

**Comment by Kromgar** [comment]
It needs the stun or how else can i be pudge?

**Comment by ExcessiveUseOfCobblestone** [comment]
@KorPhaeron are you saying it should be able to "snipe"? I don't care either way since it's rare enough that I don't think people should be upset with it [ask a miner for a conscription kit if you want the loot as well :)],  I'm just unsure why he'd nerf the stun when he doesn't even mention the stun, he mentions the range.

**Comment by Cyberboss** [comment]
The range isn't as much as a problem as the stun pull IMO

**Comment by KorPhaeron** [comment]
Well there isn't much point in pulling someone to melee them if they just walk away after

**Comment by Cyberboss** [comment]
Weaken 2 is equivalent of being shot by a dis-abler however many times so I think it's fair

**Comment by Armhulen** [comment]
>Well there isn't much point in pulling someone to melee them if they just walk away after

**Comment by optimumtact** [comment]
Just meat hook them again scrub

**Comment by ExcessiveUseOfCobblestone** [comment]
it has a CD

**Pull request closed without merging**